internal/proto/controller/storage/auth/oidc/store/v1/oidc.proto

```diff
@@ -163,6 +163,15 @@ message AuthMethod {
     this: "AccountClaimMaps"
     that: "attributes.account_claim_maps"
   }];
+
+  // prompts are the optional prompts allowed for an oidc auth method.
+  // Thee value specifies whether the authorization server prompts
+  // the end-user for reauthentication, account selection and consent
```

Comment:
Perhaps add: `These are Value Objects that will be stored as Prompt messages,
and are operatated on as a complete set.`